examples/chip-tool/BUILD.gn

```diff
@@ -76,6 +76,7 @@ static_library("chip-tool-utils") {
     "${chip_root}/src/controller/ExamplePersistentStorage.h",
     "commands/pairing/CloseSessionCommand.cpp",
     "commands/pairing/CloseSessionCommand.h",
+    "commands/pairing/DiscoverDevicesCommand.cpp",
```

Comment:
We already have a `DiscoverCommissionablesCommand`.  How does this one differ, fundamentally?